src/net/sourceforge/kolmafia/textui/command/TimeoutCommand.java

```diff
@@ -0,0 +1,15 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import net.sourceforge.kolmafia.RequestThread;
+import net.sourceforge.kolmafia.request.LogoutRequest;
+
+public class TimeoutCommand extends AbstractCommand {
+  public TimeoutCommand() {
+    this.usage = " - log out and log in again, preserving character state";
```

Comment:
just a log out here I think

Reply:
Right. "log out, leaving GUI and character state intact"
The intent is to distinguish this from the "logout" command.